docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  frontend:
+    build: ./frontend
+    volumes:
+      - ./frontend:/usr/local/src
+      - node_modules:/usr/local/src/node_modules
+    working_dir: /usr/local/src
+    entrypoint: yarn
+    command: start
```

Comment:
In case you haven't used Node.js before, `yarn` is equivalent to `npm`. In addition, when you call `yarn` or `npm` with an argument that the CLI doesn't recognize (in this case, `start`), it will read the `scripts` section of the `package.json` and call the commands defined there.

Reply:
What do you think about adding a brief comment that explains the `start` command? Mainly, a quick way to direct someone from `docker-compose.yml` to `frontend/package.json`.